ingeniamotion/fsoe.py

```diff
@@ -8,6 +8,7 @@
     DictionaryItem,
     DictionaryItemInputOutput,
     MasterHandler,
+    State,
```

Comment:
Somehow ingeniamotion should also expose the module's states, right?

The fsoe_master State is not an enum. Each state is an inheritance to the State, as it has many methods for each...

If you prefer you can map it to a ingeniamotion.fsoe enum and leave fsoe_master as a completely private dependency

Reply:
Done